api/src/wfl/source.clj

```diff
@@ -270,9 +270,11 @@
                           (find-new-rows source)
                           (create-snapshots source utc-now))]
     (when (seq shards->jobs)
-      (log/info (format "%s Snapshots created from new rows in %s.%s." (log-prefix source) (:name dataset) table))
-      (write-snapshots-creation-jobs source utc-now shards->jobs)
-      (update-last-checked source utc-now))))
+      (log/info (format "%s Snapshots created from new rows in %s.%s."
+                        (log-prefix source) (:name dataset) table))
+      (write-snapshots-creation-jobs source utc-now shards->jobs))
+    ;; Even if our poll did not yield new rows to snapshot, at least we tried:
```

Comment:
I found where we originally updated this only on snapshot creation:

https://broadinstitute.atlassian.net/browse/GH-1338
https://github.com/broadinstitute/wfl/pull/420

But I think with us now capturing missed rows, and a 20 min default interval, the considerations are no longer the same.